aldryn_redirects/validators.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, print_function, division
+import re
+
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+from six.moves.urllib.parse import urlparse
+
+
+def validate_inbound_route(value):
+    value = value.strip()
+    parsed_value = urlparse(value)
+
+    if parsed_value.scheme or parsed_value.netloc:
+        raise ValidationError(_('Do not provide the domain. This will be done automatically based on current site.'))
+
+    if parsed_value.query:
+        raise ValidationError(_('Do not provide the query params. Use "Query params" below instead.'))
+
+    if not(value.startswith('/')):
+        raise ValidationError(_('Start this field with a slash.'))
+
+    if value.endswith('/'):
+        raise ValidationError(_('Do not append a trailing slash.'))
+
+    return value
+
+
+def validate_outbound_route(value):
+    value = value.strip()
+
+    if not re.match(r'^(http://|https://|/)', value):
```

Comment:
I suggest to use something like `re.compile('^/?[a-zA-Z0-9_.-]+(/[a-zA-Z0-9_.-]+)*/?$')` (from the cms).
The reason being that this wouldn't validate things like  `/<script>hello</script>`

Reply:
I did that in an separate conditional for better error message.

```python
    if not re.match(r'^[a-zA-Z0-9\.\_\-\:\/\?\#\&\=\%]+$', value):
        raise ValidationError(_('Invalid URL provided (invalid characters found).'))
```

I added some other allowed characters (such as `&`and `?` and `=` for query params). Let me know if you think this regex is good enough.

```